deploy/osd-route-monitor-operator/100-openshift-route-monitor-operator.console.RouteMonitor.yaml

```diff
@@ -8,4 +8,4 @@ spec:
     name: console
     namespace: openshift-console
   slo:
-    targetAvailabilityPercent: "99.5"
+    targetAvailabilityPercent: "99.95"
```

Comment:
Why now 99.95? According to the [SLO doc](https://docs.google.com/document/d/1MIh_4GNTFyMaGnPBrq_cxKIrWoKw7CMShq47qp5Hj48/edit#) it should be 99.5? Probably I miss some discussion here...

Reply:
turns out this was confused with the API server SLA. We'll change it back to 99.5.